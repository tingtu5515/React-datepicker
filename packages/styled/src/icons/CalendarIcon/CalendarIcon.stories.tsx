import React from 'react'
import {storiesOf} from '@storybook/react'
import {text, color} from '@storybook/addon-knobs'
import CalendarIcon from '.'

storiesOf('Components/Icons', module).add('CalendarIcon', () => (
  <CalendarIcon
    color={color('Color', '#000')}
    height={text('Height', '30px')}
    width={text('Width', '30px')}
  />
))
