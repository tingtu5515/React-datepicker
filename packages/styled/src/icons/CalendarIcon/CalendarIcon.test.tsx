import React from 'react'
import {render} from '@testing-library/react'
import CalendarIcon from '.'

test('should match snapshot', () => {
  const {container} = render(<CalendarIcon height="30px" width="30px" color="red" />)
  expect(container).toMatchSnapshot()
})

test('should get classname', () => {
  const {container} = render(
    <CalendarIcon height="30px" width="30px" color="red" className="test" />,
  )
  expect(container).toMatchSnapshot()
})
