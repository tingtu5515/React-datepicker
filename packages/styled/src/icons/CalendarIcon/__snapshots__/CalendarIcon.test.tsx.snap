// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div>
  <svg
    color="red"
    height="30px"
    viewBox="0 0 12 12"
    width="30px"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M8 1H7v1h1V1zM6.5 6.5h-1a.5.5 0 0 0 0 1h1a.5.5 0 0 0 0-1zM6 3a.5.5 0 0 0 .5-.5v-2a.5.5 0 0 0-1 0v2A.5.5 0 0 0 6 3zm3.5 5.5h-1a.5.5 0 0 0 0 1h1a.5.5 0 0 0 0-1zm0-2h-1a.5.5 0 0 0 0 1h1a.5.5 0 0 0 0-1zM9 3a.5.5 0 0 0 .5-.5v-2a.5.5 0 0 0-1 0v2A.5.5 0 0 0 9 3zm-.5 2.5h1a.5.5 0 0 0 0-1h-1a.5.5 0 0 0 0 1zm-3 0h1a.5.5 0 0 0 0-1h-1a.5.5 0 0 0 0 1zm-2 3h-1a.5.5 0 0 0 0 1h1a.5.5 0 0 0 0-1zM11 1h-1v1h1v9H1V2h1V1H1a1 1 0 0 0-1 1v9a1 1 0 0 0 1 1h10a1 1 0 0 0 1-1V2a1 1 0 0 0-1-1zM3.5 6.5h-1a.5.5 0 0 0 0 1h1a.5.5 0 0 0 0-1zM5 1H4v1h1V1zm1.5 7.5h-1a.5.5 0 0 0 0 1h1a.5.5 0 0 0 0-1zm-4-3h1a.5.5 0 0 0 0-1h-1a.5.5 0 0 0 0 1zM3 3a.5.5 0 0 0 .5-.5v-2a.5.5 0 0 0-1 0v2A.5.5 0 0 0 3 3z"
      fill="currentColor"
      fill-rule="nonzero"
    />
  </svg>
</div>
`;
